src/instructlab/data/generator/utils.py

```diff
@@ -17,7 +17,7 @@
 
 # First Party
 from instructlab.helpers import DEFAULT_API_KEY, DEFAULT_MODEL_OLD
-from instructlab.utils import get_sysprompt, get_ssl_cert_config
+from instructlab.utils import get_ssl_cert_config, get_sysprompt
```

Comment:
It would be nicer and easier to review for all lining fixes like this to be done along the way within each commit